src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device.Mcp23xxx
```

Comment:
I'm not sure what we should do with this namespace. I don't like the fact of having Iot.Device.<someName>.<someName> types as the name become redundant, but I also don't want to have general types (like your Bank type for example) on the IoT.Device namespace. I guess we can leave it as it is for now given that you plan on coming back to this code for the bit-banging support and we can fix it then.

Reply:
Sounds good.  I struggled with this, as well.  It also falls in the bucket a little where I mentioned the "vendors" namespace issue.